Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -27,6 +27,7 @@ internal static Result CreateFolderResult(string title, string subtitle, string
                 Title = title,
                 IcoPath = path,
                 SubTitle = subtitle,
+                AutoCompleteText = $"{subtitle}\\",
```

Comment:
subtitle is probably same as path, but maybe use path is better?
```suggestion
                AutoCompleteText = $"{path}\\",
```